docs/020 - initialize-sdk-go.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Initialize SDK"
+slug: "initialize-sdk-go"
+hidden: false
+createdAt: "2019-08-21T21:13:33.317Z"
+updatedAt: "2019-08-21T21:13:45.319Z"
+---
+To initialize the **OptimizelyClient** you will need either the SDK key or hard-coded JSON datafile.
+[block:code]
+{
+  "codes": [
+    {
+      "code": "import \"github.com/optimizely/go-sdk/optimizely/client\"\n\noptimizelyFactory := &client.OptimizelyFactory{\n          SDKKey: \"[SDK_KEY_HERE]\",\n          Datafile: []byte(\"DATAFILE_JSON_STRING_HERE\")\n}\n\n// Instantiates a static client (no datafile polling)\noptlyClient, err := optimizelyFactory.StaticClient()\n\n// Instantiates a client that syncs the datafile in the background\noptlyClient, err := optimizelyFactory.Client()\n\n",
```

Comment:
Heh this is super hard to read on Github. Do you use a formatter locally to verify the structure and visual aspect of this code block? Was wondering if we can instead use GH markdown or some more commonly accepted MD format

Reply:
@mikeng13 yes, I think we talked about this earlier; I recommended that you get someone (I think you mentioned farming out to a contractor, maybe?) to migrate these docs from ReadMe MD to GH markdown, for all languages.